geomesa-accumulo/geomesa-accumulo-datastore/src/main/scala/org/locationtech/geomesa/accumulo/data/AccumuloDataStore.scala

```diff
@@ -236,6 +267,14 @@ class AccumuloDataStore(val connector: Connector, override val config: AccumuloD
       lock.release()
     }
   }
+
+  override def dispose(): Unit = {
+    super.dispose()
+    kerberosRenewerThread.map( t => {
```

Comment:
As a picky/style note, it is a convention to prefer 'foreach' over 'map' when the lambda is being used for its side-effects like here and line 90.